### ISSUE TITLE
refactor out the 62 panics in the codebase

### ISSUE THREAD

**Issue by eyberg** [nanovms/ops#344]
i didn't realize there was this much in here - in general panics should be avoided - almost every single error that is going to kill an operation for the user should be telling the user *why* it is dying

```
grep -R panic * | wc -l
62
```

**Comment by ssudake21** [comment]
@eyberg  How to reproduce this?

**Comment by eyberg** [comment]
this isn't one specific tkt - there are just a ton of places where it panics w/no meaningful error msg - so it's more of a request to be nicer to the user and improve the dev ux ->

for instance ->

```
lepton/manifest.go:             panic(err)
lepton/package.go:                      panic(err)
lepton/package.go:              panic(err)
lepton/package.go:              panic(err)
lepton/package.go:              panic(err)
lepton/package.go:              panic(err)
lepton/package.go:                      panic(err)
lepton/package.go:                                      panic(err)
lepton/package.go:                              panic(err)
lepton/package.go:                              panic(err)
lepton/const.go:                panic(err)
lepton/qemu.go:         panic(err)
lepton/image.go:                panic(err)
lepton/image.go:                panic(err)
lepton/image.go:                panic(err)
lepton/image.go:                panic(err)
lepton/image.go:                panic(err)
lepton/image.go:                panic(err)
lepton/gcp.go:          panic("We should never reach here")
Binary file ops matches
➜  ops git:(master) grep -R panic * | grep -v vendor | wc -l
      62
```

**Comment by Truenya** [comment]
Can i take this?

**Comment by eyberg** [comment]
sure - generally speaking there'd be 2 ways of removing these:

1) if it's a recoverable error just change it to ```log.Error(err)```

2) if is truly something that we wish to stop executing the program with then ->
(and preferably convert the error msg into something a normal user would somewhat understand - that' the real issue here is the program blows up and the end-user has no clue why w/out diving into the code)

```
fmt.Println(err)
os.Exit(1)
```

**Comment by Truenya** [comment]
Nice, thanks.
2 - Probably on the face of it looks like log.Fatal(err) ?

**Comment by eyberg** [comment]
yes, fatals present the same issue as panic, it's a great way to exit the code from a developer working in the codebase perspective, not so friendly to the average (espc. non-go) user

**Comment by Truenya** [comment]
Created separate package printer(in package log has duplicates, in package cmd - dependency cycle happens), with functions Infof, Debugf, Warningf, Errorf, Fatalf, printing like in package log but in command line with nice color.
Is it ok here?

**Comment by francescolavra** [comment]
Resolved by #1417
